pkg/cli/mirror/manifests.go

```diff
@@ -210,6 +210,27 @@ func generateCatalogSource(name string, dest reference.DockerImageReference) ([]
 	return cs, nil
 }
 
+func generateUpdateService(name string, releaseRepo, graphDataImage reference.DockerImageReference) ([]byte, error) {
+
+	obj := map[string]interface{}{
```

Comment:
Is there an object type somewhere for `UpdateService` in openshift library that we can use instead of having to manually construct it like this?

Reply:
AFAIK there is not, but I will do some research to double-check this.

Reply:
Was able to incorporate this, but used the `UpdateServiceSpec` type in my own type so the manifest was not generated the status fields.